Client/Frontend/Strings.swift

```diff
@@ -3565,7 +3525,7 @@ extension String {
         comment: "Message in error prompt explaining why the URL is invalid.",
         lastUpdated: .unknown)
     public static let SendToCloseButton = MZLocalizedString(
-        "SendTo.Cancel.Button",
+        "SendTo.Close.Button",
```

Comment:
Changing the key will trigger a new string on export (aka we'll loose the current translation). Same comment applies to "Settings.DisplayTheme.StatusTitle", so wanted to make sure this is what we want.

Reply:
That's exactly what we want. We had 2 strings with this ID, the export can only deal with one (per warning in the log).

In the XLIFF file we only end up with this

```
<trans-unit id="SendTo.Cancel.Button" xml:space="preserve">
        <source>Cancel</source>
        <target>Cancel</target>
        <note>Button title for cancelling share screen
   Close button in top navigation bar</note>
      </trans-unit>
```